### PR TITLE
Create a new file for interns to write tests

### PR DESCRIPTION
**Pull request by lo9key**
Create a new directory and file for interns to write tests:

/spec/utility/util_spec.rb

The idea is that we get one mostly blank test file up so we can all branch off from it with less merge conflicts.

### CODE REVIEW

**Comment by lo9key** [comment]
@Kennric

**Comment by Kennric** [comment]
@ElijahCaine @alxngyn - context: I'd like to implement a file for utility methods like averaging values or applying other standard business logic to data from the plugin report methods. Starting the interns off on testing by having them write some tests for an average method

File: spec/util/util_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require File.expand_path '../../spec_helper.rb', __FILE__
+
+describe 'The utility tests' do
+  def app
+    Iam
+  end
+
+  include Rack::Test::Methods
+
+  def app
```
Comment (alxngyn) [inline spec/util/util_spec.rb]: You have `app` defined twice here. You should probably delete one

**Comment by alxngyn** [comment]
Okay, awesome. 
You guys should look at our `/spec` folder for our current tests models and there's always documentation at http://rspec.info/documentation/3.4/rspec-core/ for Rspec to see what methods are available for you to use.

**Comment by lo9key** [comment]
@alxngyn Thanks for the heads up on that duplicate line.

**Comment by ElijahCaine** [comment]
@lo9key Awesome, this sounds like a really good idea.  There are already a few utilities that are untested like the cache and I'm sure a few more will pop up as we develop further. Good luck and don't forget to ask questions :smile:

**Comment by ElijahCaine** [comment]
@lo9key this looks like good skeleton code, should be easy to add tests too and prevent some merge conflicts.

:+1: from me

**Pull request merged**